spec/directions/bake_chapter_key_concepts_spec.rb

```diff
@@ -5,15 +5,22 @@
 RSpec.describe Kitchen::Directions::BakeChapterKeyConcepts do
   let(:chapter) do
     chapter_element(
-      page_element(
-        <<~HTML
+      <<~HTML
+        <div data-type="page">
```

Comment:
is page_element not working for you? I think it's better to use it below as well